hooks/pre/30-upgrade.rb

```diff
@@ -59,6 +67,24 @@ def remove_nodes_distributors
   Kafo::Helpers.execute("mongo pulp_database --eval  'db.repo_distributors.remove({'distributor_type_id': \"nodes_http_distributor\"});'")
 end
 
+def fix_pulp_httpd_conf
+  return true unless File.exist?('/etc/httpd/conf.d/pulp.conf.rpmnew')
+
+  Kafo::Helpers.execute('cp /etc/httpd/conf.d/pulp.conf.rpmnew /etc/httpd/conf.d/pulp.conf')
+end
```

Comment:
Does this seem idempotent enough given upgrades are run between z-releases as well?